src/bucketd/processor.rs

```diff
@@ -574,6 +574,52 @@ impl Runtime {
 
         Ok(TransferFinalize { consignment, psbt })
     }
+
+    pub(super) fn finalize_transfers(
+        &mut self,
+        transfers: Vec<(StateTransfer, Vec<SealEndpoint>)>,
+        mut psbt: Psbt,
+    ) -> Result<FinalizeTransfersRes, DaemonError> {
+        // 1. Pack LNPBP-4 and anchor information.
+        let mut bundles = psbt.rgb_bundles()?;
+        debug!("Found {} bundles", bundles.len());
+        trace!("Bundles: {:?}", bundles);
+
+        let anchor = Anchor::commit(&mut psbt)?;
+        trace!("Anchor: {:?}", anchor);
+
+        let mut consignments = vec![];
+        for transfer in &transfers {
+            let mut consignment = transfer.0.clone();
```

Comment:
You should destructure this like so:
```rust
for (state_transfer, seal_endpoints) in &transfers {
    let mut consignment = state_transfer.clone();
    // ...
    let endseals = seal_endpoints.clone();
    // ...
}
```
Or something like that. It's more self-documenting.

Reply:
Hi @hunter,

I've just noticed this comment, sorry. I've updated the code with your suggestion, thanks.

> Also, I'm having trouble understanding how `Vec<(StateTransfer, Vec<SealEndpoint>)>` gets defined.

Not sure what you mean. That parameter represents the list of consignments to finalize, each consignment is associated with a list of seal endpoints (the beneficiaries of that asset transfer).